.github/workflows/lighthouse-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: Lighthouse CI
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install and Build
+        run: |
+          yarn install
+          yarn build
+      - name: Wait for the Netlify Preview
+        uses: jakepartusch/wait-for-netlify-action@v1
+        id: netlify
+        with:
+          site_name: 'keptn-experimental-docs-site'
+      - name: Audit URLs using Lighthouse
+        id: lighthouse_audit
+        uses: treosh/lighthouse-ci-action@v3
+        with:
+          urls: |
+            https://deploy-preview-$PR_NUMBER--keptn-experimental-docs-site.netlify.app/
```

Comment:
any reason why we're just sending two URLs? Is it useful to send all the pages to lighthouse?

Reply:
Yeah, it's good to send both the URLs i.e. the URL where the website is hosted and the URL of the page which load first when we open the website.

Reply:
Hi Rajiv, I meant, is there a way to send all the pages of docs to lighthouse, instead of just 2?